runtime/autoload/remote/host.vim

```diff
@@ -190,6 +190,7 @@ function! s:RegistrationCommands(host) abort
   call remote#host#RegisterClone(host_id, a:host)
   let pattern = s:plugin_patterns[a:host]
   let paths = globpath(&rtp, 'rplugin/'.a:host.'/'.pattern, 0, 1)
+  let paths = map(paths, 'resolve(v:val)')
   let paths = map(paths, 'tr(v:val,"\\","/")') " Normalize slashes #4795
```

Comment:
can you add the resolve() call in here? the "let paths" is getting a bit too verbose